README.md

```diff
@@ -33,9 +33,6 @@ const saml = new SAML(options);
 
 - **Core**
 - `callbackUrl`: full callbackUrl (overrides path/protocol if supplied)
```

Comment:
I was considering that, and I see your point, but I hesitated because it would have created dozens of new lines of code in the tests... but I suppose that would make sense: force people to think about what they are doing for a security application. @markstos , what do you think?